src/scanners/css.js

```diff
@@ -31,25 +31,23 @@ export default class CSSScanner extends BaseScanner {
     }
 
     var file = cssCode.position.source;
-    var startLine = cssCode.position.start.line;
-    var startColumn = cssCode.position.start.column;
+    var options = Object.assign(this.options, {
```

Comment:
It might be better to call this something else and the use `Object.assign({}, this.options, { blah })`